CMakeLists.txt

```
@@ -1,50 +1,82 @@
-cmake_minimum_required(VERSION 3.0.2)
+cmake_minimum_required(VERSION 3.8 FATAL_ERROR)
```

```suggestion
cmake_minimum_required(VERSION 3.8)
```
あまり使っているのを見たことがないんですが、`FATAL_ERROR`が必要な理由はありますか？

---

特に理由はないです．
エラーが出たときに修正したときに間違えて追加したものだと思います．

---

不要であれば削除してください